CHANGELOG.md

```diff
@@ -25,6 +26,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Fixed
 
 - Tray: remove `version` immutable field in DC selector
+- Tray: remove OKD compatibility
```

Comment:
Should be in the _removed_ section?

Reply:
Yes and both of them, no?

Reply:
I'd merge the 2 lines into one and put it in the `Changed` section :sweat_smile: